gcp_variant_transforms/libs/variant_merge/merge_with_non_variants_strategy_test.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for merge_with_nonvariants_strategy."""
+
+from __future__ import absolute_import
+
+import unittest
+
+
+from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms.libs.bigquery_vcf_schema import ColumnKeyConstants
+from gcp_variant_transforms.libs.variant_merge import (
```

Comment:
nit: long import statements are actually exempt from line length, so you can keep it as one line.

Reply:
Done.